exec/plugin-commands-script-runners/test/index.ts

```diff
@@ -465,3 +465,72 @@ test('pnpm run with custom shell', async () => {
 
   expect((await import(path.resolve('shell-input.json'))).default).toStrictEqual(['-c', 'foo bar'])
 })
+
+test('pnpm run with multiple script selector should work', async () => {
+  prepare({
+    scripts: {
+      'build:a': 'node -e "process.stdout.write(\'a\')" | json-append ./output-a.json',
+      'build:b': 'node -e "process.stdout.write(\'b\')" | json-append ./output-b.json',
+      'build:c': 'node -e "process.stdout.write(\'c\')" | json-append ./output-c.json',
+    },
+  })
+
+  await execa('pnpm', ['add', 'json-append@1'])
```

Comment:
why do you need to use json-append for these tests? json-append is useful if you want to check the sequence of the scripts. You can append items to the same file. But you use different files, so you can just create empty files.

like 

```
'build:c': 'node -e "require('fs').writeFileSync('a', '', 'utf8')"',
```

Reply:
Certainly, I used json-append just because other unit tests use this. b606268 :+1: